frontend/src/ItemView.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Button,
+  Card,
+  CardContent,
+  Container,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+import React, { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Item } from "./util/util-types";
+import ImageGallery from "react-image-gallery";
+import "react-image-gallery/styles/scss/image-gallery.scss";
+import styles from "./css/ItemView.module.css";
+
+type Props = {
+  readonly loggedIn: boolean;
+};
+export default function ItemView({ loggedIn }: Props) {
+  const { item_id } = useParams<Record<string, string | undefined>>();
+  const history = useHistory();
+  const [item, setItem] = useState<Item>({
+    name: "bruh",
+    brand: "bruh",
```

Comment:
use better default state values

Reply:
it's fine to use `null` and stuff even if it'll get overwritten really quickly

Reply:
Also render a loading animation (https://material-ui.com/components/progress/) if the fetch hasn't completed yet i.e. the state is unchanged from the original